src/main/scala/net/psforever/packet/game/GenericActionMessage.scala

```diff
@@ -2,76 +2,69 @@
 package net.psforever.packet.game
 
 import net.psforever.packet.{GamePacketOpcode, Marshallable, PlanetSideGamePacket}
+import enumeratum.values.{IntEnum, IntEnumEntry}
 import scodec.Codec
 import scodec.codecs._
 
+sealed abstract class GenericAction(val value: Int) extends IntEnumEntry
+
+object GenericAction extends IntEnum[GenericAction] {
+  val values: IndexedSeq[GenericAction] = findValues
+
+  final case object ShowMosquitoRadar extends GenericAction(value = 3)
+  final case object HideMosquitoRadar extends GenericAction(value = 4)
+  final case object MissileLock extends GenericAction(value = 7)
+  final case object WaspMissileLock extends GenericAction(value = 8)
+  final case object TRekLock extends GenericAction(value = 9)
+  final case object DropSpecialItem extends GenericAction(value = 10)
+  final case object FacilityCaptureFanfare extends GenericAction(value = 12)
+  final case object NewCharacterBasicTraining extends GenericAction(value = 14)
+  final case object MaxAnchorsExtend extends GenericAction(value = 15)
+  final case object MaxAnchorsRelease extends GenericAction(value = 16)
+  final case object MaxSpecialEffect extends GenericAction(value = 20)
```

Comment:
The previous comment had information about if the action should be from the client, or from the server to be effective. Is it worth prefixing these with something like `Server`/`Client`?

Reply:
I'm not confident if that kind of organization was ever useful.  In the beginning, I just parsed ordered packet data that was just a bunch of numbers, separated only by the direction they were encountered, since they were just numbers, then tried to attach meaning to them through trial and error.  That's how the original list was created,  Now that they have names, I don't know if they need direction designation.
Maybe the ones that come from the client, as the majority of them are still packets you send to the server.